maps/randomvaults/objects.dm

```diff
@@ -982,4 +982,12 @@
 
 /area/vault/mini_station_construction
 	name = "NT Microstation Construction Room"
-	icon_state = "construction"
\ No newline at end of file
+	icon_state = "construction"
+
+/obj/item/device/pda/clown/broken
+	name = "Antique Clown PDA"
+	desc = "A portable microcomputer by Thinktronic Systems, LTD. The surface is coated with polytetrafluoroethylene and banana drippings. This one has been stepped on for too many times, and appears to be completely unresponsible."
```

Comment:
unresponsible

Reply:
Description was copied over from the original edit